README.md

```diff
@@ -56,49 +56,32 @@ conda install orange3-<addon name>
 ```
 [See specific add-on repositories for details.](https://github.com/biolab/)
 
-#### Running
-
-Activate the environment, 
-```Shell
-conda activate orange3
-``` 
-and run either `orange-canvas` or `python3 -m Orange.canvas`. 
-
-Add `--help` for a list of program options.
-
-Starting up for the first time may take a while.
 
 ### Installing with pip
 
-To install Orange with pip, run the following.
+We recommend using our [standalone installer](https://orange.biolab.si/download) or conda, but Orange is also installable with pip. On Windows, you will also need Microsoft Visual Studio Build Tools.
```

Comment:
You will need a C/C++ compiler (on Windows we suggest using Microsoft Visual Studio Build Tools).